packages/components/src/styles/media/media.stories.ts

```diff
@@ -35,25 +39,21 @@ export const Default = {
   }
 };
 
-export const MediaSample = {
-  parameters: {
-    controls: {
-      disable: true
-    }
-  },
-  render: (args: any) => {
-    return generateTemplate({
-      options: {
-        templateContent: `
-          <figure class="%CLASSES% max-w-xl p-4">
-            <div class="sd-copyright" style="--copyright: '© 2024 Solid Design System';">
-              <img src="./placeholders/images/generic.jpg" alt="A generic placeholder jpg" class="aspect-video object-cover"/>
-            </div>
-            <figcaption>%SLOT%</figcaption>
-          </figure>
-      `
-      },
-      args
-    });
-  }
+/**
+ * Use the `&--inverted` class when displayed on primary background.
+ */
```

Comment:
In figma we have 2 more sections:
- Description
- Copyright

Can we have them?

Reply:
They are in the template